src/app/pages/website/about-us/about-us.component.html

```diff
@@ -1,43 +1,15 @@
 <littil-title>Over ons</littil-title>
 
 <littil-content-container>
-    <p>Project LITTIL is ontstaan uit een idee van Stichting Devoxx4Kids Nederland. Deze stichting heeft
-      meerdere
-      aanvragen van scholen gehad om een gastles te komen verzorgen.
-    </p>
-    <p>
-      Wij willen deze scholen graag helpen om een gast docent te vinden in hun omgeving die aansluit op hun wensen. Daarom
-      zijn wij gestart met 3 scrum teams en 60 vrijwilligers die hier onderzoek naar doen.
-    </p>
+  <p>Project LITTIL is ontstaan uit een idee van Stichting Devoxx4Kids Nederland. Deze stichting heeft
+    meerdere
+    aanvragen van scholen gehad om een gastles te komen verzorgen.
+  </p>
+  <p>
+    Wij willen deze scholen graag helpen om een gast docent te vinden in hun omgeving die aansluit op hun wensen.
+  </p>
 
-    <h2>Scrumteam Scholen</h2>
-    <p>Binnen Scrumteam Scholen is Hans Lieben Scrum Master. Hij zorgt ervoor dat zijn team informatie haalt
-      bij scholen
-      en docenten. Denk hierbij aan wat scholen nodig hebben om een match te vinden binnen onze lijst van
-      gastdocenten.
-    </p>
-
-    <h2>Scrumteam Gastdocenten</h2>
-    <p>
-      Binnen Scrumteam Gastdocenten is Jasper Sprengers Scrum Master. Hij zorg ervoor dat zijn team informatie haalt bij
-      toekomstige gast docenten. Denk hierbij aan welke lessen zij willen geven en in welke regio.
-    </p>
-
-    <h2>Scrumteam Marketing / Marktonderzoek</h2>
-    <p>
-      Binnen Scrumteam Markerting / Marktonderzoek is Eddy Vos Scrum Master. Hij zorgt ervoor dat zijn team onderzoek doet
-      naar mogelijke partners die dit platform kunnen versterken. Denk hierbij aan sponsors die hun bedrijfsnaam graag
-      willen koppelen aan dit initiatief. Of aan andere organisaties die lesmodules verzorgen maar geen docenten hebben.
-    </p>
-
-    <h2>Alle hulp is welkom</h2>
```

Comment:
dit is de tekst review change van @MarcelWildenburg , ook verplaatst naar littil-contact-banner